.github/workflows/ci.yml

```diff
@@ -48,19 +48,31 @@ jobs:
   should_run:
     runs-on: ubuntu-20.04
     env:
-      SHOULD_RUN: ${{ github.event_name != 'pull_request' || ! contains(github.event.pull_request.labels.*.name, 'skip-ci') }}
+      BASE_REF: HEAD^
+      LABELS: join(${{ github.event.pull_request.labels.*.name }})
     outputs:
-      should-run: ${{ env.SHOULD_RUN }}
+      should-run: ${{ steps.should-run.outputs.should-run }}
     steps:
+      - name: "Checking out repository"
+        uses: actions/checkout@7884fcad6b5d53d10323aee724dc68d8b9096a2e # v2
+        with:
+          # We need the parent commit to do a diff
+          fetch-depth: 2
```

Comment:
I realized that we can do this with all our lint actions that are manually fetching "main" too